index.js

```diff
@@ -74,13 +74,24 @@ Remit.prototype.res = function res (event, callbacks, context, options) {
         self.__assert_exchange(() => {
             const chosen_queue = options.queueName || event
 
+            const queueOptions = {
```

Comment:
With these options, can we change the check to be `options.hasOwnProperty('xyz')` and then, if `true`, coerce the value to a boolean using `!!`?